crates/mapi-sys/CHANGELOG.md

```diff
@@ -1,26 +1,42 @@
-# Changelog
```

Comment:
In both change logs, it looks like maybe there was a whitespace/newline change on these lines. Running `release-plz update` duplicated all of them and inserted the changes for the next release. I manually edited it to remove the duplicated content.